samples/web/content/apprtc/README.md

```diff
@@ -60,4 +60,33 @@ To run only the Python tests you can call,
 grunt shell:runPythonTests
 ```
 
+### Enabling Local Logging
+
+By default, logging to a BigQuery from the development server is disabled. Log information is presented on the console. Unless you are modifying the analytics API you will not need to enable remote logging. Nothing is required when running on on Google App Engine.
+
+Logging to BigQuery when running LOCALLY requires a `secrets.json` containing Service Account credentials to a Google Developer project where BigQuery is enabled. DO NOT COMMIT `secrets.json` TO THE REPOSITORY.
+
+To generate a `secrets.json` file in the Google Developers Console for your project:
+1. Go to the project page.
+1. Under *APIs & auth* select *Credentials*.
+1. Confirm a *Service Account* already exists or create it by selecting *Create new Client ID*.
+1. Select *Generate new JSON key* from the *Service Account* area to create and download JSON credentials.
+1. Rename the downloaded file to `secrets.json` and place in the directory containing `analytics.py`.
```

Comment:
I mention that nothing is required when working on Google App Engine. Please let me know if the wording can be improved. I will move the line to the top.